gen-api-docs.bash

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-lemmy-help \
+lemmy-help -fact \
```

Comment:
What does `-fact` do?

Reply:
-f, --prefix-func       Prefix function name with ---@mod name
    -a, --prefix-alias      Prefix ---@alias tag with return/---@mod name
    -c, --prefix-class      Prefix ---@class tag with return/---@mod name
    -t, --prefix-type       Prefix ---@type tag with ---@mod name

I don't think all of them makes a difference atm, but I added all of them just to be future-proof

Reply:
Got it. That makes sense. Thanks.